torch/_subclasses/fake_tensor.py

```diff
@@ -1307,14 +1307,20 @@ def dispatch(self, func, types, args=(), kwargs=None):
                 if op_impl_out != NotImplemented:
                     return op_impl_out
 
+        def can_fallback(func: OpOverload):
+            if not self.allow_fallback_kernels:
+                return False
+            # should't fallback on custom ops
```

Comment:
nit: leave a comment with some more context? Something like "It's OK to try the fallback for built-in ops (e.g. aten, prims) because we control and test these but the fallback leads to unexpected behavior in user-defined custom ops"